components/match2/commons/starcraft_starcraft2/match_group_input_starcraft.lua

```diff
@@ -72,24 +71,16 @@ function StarcraftMatchGroupInput.processMatch(match, options)
 end
 
 function StarcraftMatchGroupInput._readDate(matchArgs)
-	if matchArgs.date then
-		local dateProps = MatchGroupInput.readDate(matchArgs.date)
-		dateProps.dateexact = Logic.nilOr(Logic.readBoolOrNil(matchArgs.dateexact), dateProps.dateexact)
+	local dateProps = MatchGroupInput.readDate(matchArgs.date, {
+		'matchDate',
+		'Match_date',
```

Comment:
Can I drop this one too @hjpalpha ? You know SC1/SC2 better than me

Reply:
sc2 would be ready to drop `'Match_date',`, sc1 not yet
i can run a bot though
so if we merge this tomorrow earliest it can be dropped

Reply:
bot run done, we can drop `'Match_date',`